drivers/serial/Kconfig

```diff
@@ -81,6 +81,11 @@ config RPMSG_UART
 	select SERIAL_REMOVABLE
 	select SERIAL_IFLOWCONTROL
 
+config RPMSG_UART_CONSOLE
+	bool "UART rpmsg console support"
+	default n
+	depends on RPMSG_UART
+
```

Comment:
Please include a help to let user know when to use this feature

Reply:
done